src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -232,7 +232,13 @@
      {:style (style/discover-screen-container (colors/theme-colors
                                                colors/white
                                                colors/neutral-95))}
-     [discover-screen-content featured-communities theme]]))
+     [discover-screen-content featured-communities theme]
+     [quo/floating-shell-button
+      {:jump-to {:on-press            #(rf/dispatch [:shell/navigate-to-jump-to])
+                 :customization-color :blue
```

Comment:
can we get the correct `customization-color` here?

Reply:
sorry leftover from debug, will fix that. Thank you